iam/__tests__/brightid.test.ts

```diff
@@ -0,0 +1,168 @@
+// --- Test subject
+import { BrightIdProvider } from "../src/providers/brightid";
+import { triggerBrightidSponsorship } from "../src/procedures/brightid";
+import { BrightIdVerificationResponse, BrightIdSponsorshipResponse } from "@dpopp/types";
+import { RequestPayload } from "@dpopp/types";
+import { verifyContextId, sponsor } from "brightid_sdk";
+
+jest.mock("brightid_sdk", () => ({
+  verifyContextId: jest.fn(),
+  sponsor: jest.fn(),
+}));
+
+describe("Attempt BrightId", () => {
+  const did = "did:3:kjzl0cwe1jw1475ckc0ib6k44mm4sqegorxc1x23ppqh9lt9quso6yp7ayh2fae";
```

Comment:
Does this need to be genuine DID (we're also not using `did:3`s anymore - this should probably be a `did:pkh` (`did:pkh:eip155:1:0x0`))?